### PR TITLE
S2i leftovers

### PR DESCRIPTION
**Pull request by soltysh**
Fixes leftovers from #275.

@mfojtik @bparees

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -11,8 +11,9 @@ import (
 	"github.com/spf13/pflag"
 )
 
-// DefaultConfigPath specifies the default location of the STI config file
-const DefaultConfigPath = ".stifile"
+// DefaultConfigPath specifies the default location of the S2I config file
+const DeprecatedConfigPath = ".stifile"
```
Comment (mfojtik) [inline pkg/config/config.go]: i'm not a fan of having these 'Deprecated*' constants... we are not doing that of other deprecated stuff and we will expose them to users as API (where there is no point exposing them).

I would have just one constant and handle deprecation below, what do you think?

**Comment by mfojtik** [comment]
@soltysh one nit, otherwise :+1: 

also please squash you cheater.

**Comment by soltysh** [comment]
Comment addressed.

> also please squash you cheater.

It's to distinguish which change is where :smile:

**Comment by mfojtik** [comment]
[merge]

On Wed, Feb 3, 2016 at 9:43 AM, Maciej Szulik <notifications@github.com>
wrote:

> Comment addressed.
>
> also please squash you cheater.
>
> It's to distinguish which change is where [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/408#issuecomment-179094582>
> .
>



-- 
-----------------------------------------------
Michal Fojtik <mi@mifo.sk <mfojtik@mifo.sk>>
Red Hat OpenShift, Engineering

**Pull request merged**